postgraphile/postgraphile/__tests__/schema/v4/defaultOptions.1.export.mjs

```diff
@@ -12373,6 +12389,15 @@ const makeArgs64 = (args, path = []) => {
   return selectArgs;
 };
 const resource_compound_type_computed_fieldPgResource = registry.pgResources["compound_type_computed_field"];
+function UUIDSerialize(value) {
+  return "" + value;
+}
```

Comment:
As an optimization we should have this export as `toString` (the source name) rather than being called `UUIDSerialize` everywhere. But no big deal.